aws/resource_aws_shield_protection.go

```diff
@@ -0,0 +1,103 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/shield"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsShieldProtection() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsShieldProtectionCreate,
+		Read:   resourceAwsShieldProtectionRead,
+		Delete: resourceAwsShieldProtectionDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceAwsShieldProtectionImport,
```

Comment:
Instead of implementing a custom import function here, we should simplify this to use `schema.ImportStatePassthrough` 👍 

```suggestion
			State: schema.ImportStatePassthrough,
```

The two `d.Set()` calls that are present in the custom import function should be moved into the `Read` function so they are refreshed every plan (and relevant to this feedback, import calls `Read`). 😄